--- scripts/services/dovecot.orig       2013-09-17 09:01:11.909280745 +0200
+++ scripts/services/dovecot    2013-09-23 08:29:55.704453859 +0200
@@ -130,12 +130,15 @@
 }

 # Handle "dovecot: <svc>" and "dovecot: [ID yyyyy mail.info] <svc"
-my $dovecottag = qr/dovecot:(?:\s*\[[^]]+\])?/;
+#my $dovecottag = qr/dovecot:(?:\s*\[[^]]+\])?/;
+# Also handle optional hostname before dovecot tag for syslog format.
+my $dovecottag = qr/(?:[a-z][a-z0-9]+ )?dovecot:(?:\s*\[[^]]+\])?/;
+

 while (defined($ThisLine = <STDIN>)) {
      # remove timestamp.  We can't use *RemoveHeaders because we need the
      # service name
-     #$ThisLine =~ s/^\w{3} .\d \d\d:\d\d:\d\d (?:[^\s:]* |)//;
+     $ThisLine =~ s/^\w{3} .\d \d\d:\d\d:\d\d (?:[^\s:]* |)//;
      if ( ($ThisLine =~ /(?:ssl-build-param|ssl-params): SSL parameters regeneration completed/) or
           ($ThisLine =~ /ssl-params: Generating SSL parameters/) or
           ($ThisLine =~ /auth-worker/) or
